awx/main/signals.py

```diff
@@ -48,15 +48,39 @@ def emit_job_event_detail(sender, **kwargs):
     created = kwargs['created']
     if created:
         event_serialized = JobEventWebSocketSerializer(instance).data
-        emit_channel_notification('job_events-' + str(instance.job.id), event_serialized)
+        consumers.emit_channel_notification('job_events-' + str(instance.job.id), event_serialized)
 
 
 def emit_ad_hoc_command_event_detail(sender, **kwargs):
     instance = kwargs['instance']
     created = kwargs['created']
     if created:
         event_serialized = AdHocCommandEventWebSocketSerializer(instance).data
-        emit_channel_notification('ad_hoc_command_events-' + str(instance.ad_hoc_command_id), event_serialized)
+        consumers.emit_channel_notification('ad_hoc_command_events-' + str(instance.ad_hoc_command_id), event_serialized)
+
+
+def emit_project_update_event_detail(sender, **kwargs):
```

Comment:
These serializes all have `get_group_name` methods that we should use here.

Reply:
Also might be nice to break out the group formatting to it's own line using `format`, but that's up to you.